modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/jdbc/ItJdbcStatementSelfTest.java

```diff
@@ -718,8 +718,9 @@ public void testStatementTypeMismatchSelectForCachedQuery() throws Exception {
     }
 
     @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-16268")
     public void testStatementTypeMismatchUpdate() throws Exception {
+        stmt.executeUpdate("update TEST set NAME='28' where ID=1");
```

Comment:
this update is hiding original problem

Reply:
Refactored the test. Test methods are now isolated from each other.